src/common/utility.js

```diff
@@ -363,8 +363,8 @@ export const getBaseRef = (ref) => {
     return ref?.current?.base?.style ? ref?.current?.base : ref?.current
 }
 
-const eu_subdomain_countries = ['nl']
 const uk_subdomain_countries = ['gb']
+const eu_subdomain_countries = ['nl']
 
```

Comment:
Let's make sure we put significant description in the commit message even if it's a very tiny change.

Reply:
Yeah sure, my bad.